build/ci-build.yml

```diff
@@ -79,10 +81,10 @@ steps:
   inputs:
     pythonUploadServiceConnection: 'Arcus ML PyPi feed'
 
-- script: |
-   pdoc --html --force --config show_source_code=False -o docs/ arcus/ml 
-  displayName: 'Generate code documentation'
-  continueOnError: true
+# - script: |
+#    pdoc --html --force --config show_source_code=False -o docs/ arcus/ml 
+#   displayName: 'Generate code documentation'
+#   continueOnError: true
```

Comment:
I guess this can be removed now?

Reply:
yes , but we still have to make sure the generated md files are compatible & can be parsed nicely with the netlify docs

Reply:
removed with new commit